python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -21,6 +21,10 @@
 from bigdl.llm.utils.common import invalidInputError
 
 
+def save_low_bit(self, *args, **kwargs):
+    self.save_pretrained(*args, **kwargs)
```

Comment:
we should remove `save_pretrained` method and only use `save_low_bit`

Reply:
If I remove save_pretrained, how can I call `save_pretrained` in save_low_bit?

Reply:
Just record, we can delete save_pretrained like this:
```python
        import types
        from functools import partial
        save_func = partial(model.save_pretrained)
        save_low_bit_func = partial(save_low_bit, save_func=save_func)  # call this instead of save_pretrained in save_low_bit
        model.save_low_bit = types.MethodType(save_low_bit_func, model)
        from transformers.modeling_utils import PreTrainedModel
        del PreTrainedModel.save_pretrained
```

Reply:
No need to delete for now

Reply:
Model loaded with load_low_bit can be saved using save_load_bit correctly.